src/Constants/ValidationConstants.cs

```diff
@@ -10,5 +10,6 @@ public class ValidationConstants
         public const string BOOKING_DATE_AND_TIME_EMPTY = "You must select a date and time";
         public const string BOOKING_DATE_EMPTY = "You must select a date";
         public const string BOOKING_TIME_EMPTY = "You must select a time";
+        public const string STREET_INCORRECT_FORMAT = "Enter the street name in the correct format";
```

Comment:
error message does not follow GOVUK recommended for invalid format
![image](https://user-images.githubusercontent.com/35955528/105689683-f75b8980-5ef2-11eb-9f23-da70bbc49cc9.png)

If this can be confirmed with content if we do not want to use govuk approach.